### PR TITLE
[SMALLFIX] Improve mini benchmark

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 1]
```diff
@@ -46,94 +61,157 @@
...
+   * Prints the usage.
+   */
   private static void usage() {
-    System.out.println("Usage:");
-    System.out.println("To run a mini benchmark to write or read a file.");
-    System.out.println(String
-        .format("java -cp %s %s <[READ, WRITE]> <fileSize> <iterations>",
-        RuntimeConstants.ALLUXIO_JAR, MiniBenchmark.class.getCanonicalName()));
+    new HelpFormatter().printHelp(
+        String.format("java -cp %s %s <[READ, WRITE]> <fileSize> <iterations>",
```
Comment (ifcharming) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: update the usage with the latest options `-type`, `-fileSize`, `-iterations` and `-concurrency`?

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 2]
```diff
@@ -46,94 +61,157 @@
...
-   * @param iterations number of iterations
+   * @param count the count to determine the filename
+   * @throws Exception if it fails to write
    */
-  private static void writeFile(long fileSize, int iterations) throws Exception {
+  private static void writeFile(int count) throws Exception {
     FileSystem fileSystem = FileSystem.Factory.get();
-    byte[] buffer = new byte[(int) Math.min(fileSize, 4 * Constants.MB)];
+    byte[] buffer = new byte[(int) Math.min(sFileSize, 4 * Constants.MB)];
     Arrays.fill(buffer, (byte) 'a');
```
Comment (ifcharming) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: FYI, i'll probably add a random context generator later here.

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java (hunk 1 above)
Comment (peisun1115) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: done

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java (hunk 2 above)
Comment (peisun1115) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: k. ty

**Pull request merged**